mappings/net/minecraft/util/math/Quaternion.mapping

```diff
@@ -1,11 +1,14 @@
 CLASS net/minecraft/class_1158 net/minecraft/util/math/Quaternion
 	FIELD field_21493 IDENTITY Lnet/minecraft/class_1158;
-	FIELD field_21585 a F
+	FIELD field_21582 a F
+	FIELD field_21583 b F
+	FIELD field_21584 c F
+	FIELD field_21585 d F
```

Comment:
hmm, we should start naming them x y z w if people are really confused. the first three are multipliers on 3 imaginary components and the last one is the multiplier on the real component. I don't know why but these mappings lost again.

Reply:
Yes, call them x, y, z, w please, that's what they're called.

Reply:
appears that some modders may agree with the xyzw names, such as https://github.com/jellysquid3/sodium-fabric/commit/fcfccc3d534c866f64c6e24e4f5407a2339cff37#diff-807d8c9858c64d22dc8ed37a70b4280cR88-R91

Reply:
done